src/ripple/app/tx/impl/AMMVote.cpp

```diff
@@ -179,7 +181,7 @@ applyVote(
         {
             auto const entry = updatedVoteSlots.begin() + minPos;
             // Remove the least token vote entry.
-            num -= Number((*entry)[sfTradingFee]) * *minTokens;
+            num -= Number((*entry)[~sfTradingFee].value_or(0)) * *minTokens;
```

Comment:
Just to note, this can NOT be changed back to the original code because `entry` is a new object, and default field handling doesn't work as it does with a parsed object.